### PR TITLE
Fast Pair documentation update bundle

### PR DESCRIPTION
**Pull request by kapi-no**

### CODE REVIEW

File: doc/nrf/external_comp/dult.rst [hunk 1]
```diff
@@ -12,7 +12,12 @@ Following the specification improves the privacy and safety of individuals by pr
 For detailed information about supported functionalities, see the official `DULT`_ documentation.
 
 .. note::
+   The `DULT`_ documentation has the Internet-Draft status.
+   Internet-Drafts are draft documents valid for a maximum of six months and may be updated, replaced, or obsoleted by other documents at any time.
```
Comment (MarekPieta) [inline doc/nrf/external_comp/dult.rst:16]: Consider explicitly marking it as a quite from `DULT status`

**Review by MarekPieta — APPROVED** [review]

File: doc/nrf/external_comp/dult.rst (hunk 1 above)
Comment (kapi-no) [reply doc/nrf/external_comp/dult.rst:16]: @annwoj, how can we mark it as a quote?

File: doc/nrf/external_comp/dult.rst [hunk 2]
```diff
@@ -12,7 +12,12 @@ Following the specification improves the privacy and safety of individuals by pr
...
+   See the `DULT status`_ section from the DULT specification for more details.
+
    The DULT support in the |NCS| is :ref:`experimental <software_maturity>`.
+   We may introduce breaking changes in the DULT support by following the changes in the `DULT`_ specification.
```
[the remark this answers is not in the document]
Comment (kapi-no) [reply doc/nrf/external_comp/dult.rst:20]: much more elegant way to say it :)

File: doc/nrf/external_comp/dult.rst (hunk 1 above)
Comment (kapi-no) [reply doc/nrf/external_comp/dult.rst:16]: oh, nvm, @annwoj  suggested to rephrase it entirely

**Comment by kapi-no** [comment]
addressed review comments

**Comment by kapi-no** [comment]
Rebased, planning to add one additional sentence in the integration guide that points to SW table.

File: doc/nrf/external_comp/bt_fast_pair.rst [hunk 3]
```diff
@@ -843,9 +843,11 @@ Tailoring protocol for specific use case
 ****************************************
 
 The specific use case of the Google Fast Pair application is indicated by the chosen device type in the Google Nearby Devices console (see the :ref:`ug_bt_fast_pair_provisioning_register_device_type` subsection).
-Different use cases may require implementation of additional guidelines for your accessory firmware or specific configuration of your device model in the Google Nearby Devices console.
-These requirements typically help to improve user experience or security properties for the chosen use case.
-To learn more, see the official `Fast Pair`_ documentation.
+The `Fast Pair Device Feature Requirements`_ category in the official `Fast Pair`_ documentation defines additional requirements for each of the supported use cases.
+Each listed use case specifies a list of mandatory and unsupported Fast Pair features.
```
Comment (MarekPieta) [inline doc/nrf/external_comp/bt_fast_pair.rst:847]: `mandatory, optional and not supported.` (e.g. see `Automotive`)

File: doc/nrf/external_comp/bt_fast_pair.rst [hunk 4]
```diff
@@ -856,7 +856,12 @@ Locator tag is a small electronic device that can be attached to an object or a
 The locator tags can use different wireless technologies like GPS, Bluetooth LE or UWB for location tracking.
 It is even possible to combine multiple technologies in a single product to improve the user experience.
 
-The FMDN Accessory specification lists `Fast Pair Locator Tag Specific Guidelines`_ for the locator tag use case.
+The `Fast Pair Device Feature Requirements for Locator Tags`_ documentation defines the Fast Pair requirements for the locator tag use case.
+If your product is targeting the locator tag use cases, you must configure your application according to these requirements.
```
Comment (MarekPieta) [inline doc/nrf/external_comp/bt_fast_pair.rst:860]: `locator tag use case`?

**Review by MarekPieta — APPROVED** [review]

File: doc/nrf/external_comp/bt_fast_pair.rst [hunk 5]
```diff
@@ -843,9 +843,11 @@ Tailoring protocol for specific use case
 ****************************************
 
 The specific use case of the Google Fast Pair application is indicated by the chosen device type in the Google Nearby Devices console (see the :ref:`ug_bt_fast_pair_provisioning_register_device_type` subsection).
-Different use cases may require implementation of additional guidelines for your accessory firmware or specific configuration of your device model in the Google Nearby Devices console.
-These requirements typically help to improve user experience or security properties for the chosen use case.
-To learn more, see the official `Fast Pair`_ documentation.
+The `Fast Pair Device Feature Requirements`_ category in the official `Fast Pair`_ documentation defines additional requirements for each of the supported use cases.
```
Comment (annwoj) [inline doc/nrf/external_comp/bt_fast_pair.rst:846]:
```suggestion
The `Fast Pair Device Feature Requirements`_ category in the official `Fast Pair`_ documentation defines additional requirements for each supported use case.
```

File: doc/nrf/external_comp/bt_fast_pair.rst [hunk 6]
```diff
@@ -843,9 +843,11 @@ Tailoring protocol for specific use case
...
+Each listed use case specifies a list of mandatory and unsupported Fast Pair features.
+If your product is targeting one of the listed use case, you should align your accessory firmware to satisfy its requirements.
```
Comment (annwoj) [inline doc/nrf/external_comp/bt_fast_pair.rst:848]:
```suggestion
If your product is targeting one of the listed use cases, you must align your accessory firmware to meet these requirements.
```

File: doc/nrf/external_comp/bt_fast_pair.rst (hunk 4 above)
Comment (kapi-no) [reply doc/nrf/external_comp/bt_fast_pair.rst:860]: this was just a copy past of another sentence so I am going to align also that one

**Review by MarekPieta — APPROVED** [review]

File: doc/nrf/external_comp/bt_fast_pair.rst [hunk 7]
```diff
@@ -843,9 +843,11 @@ Tailoring protocol for specific use case
 ****************************************
 
 The specific use case of the Google Fast Pair application is indicated by the chosen device type in the Google Nearby Devices console (see the :ref:`ug_bt_fast_pair_provisioning_register_device_type` subsection).
-Different use cases may require implementation of additional guidelines for your accessory firmware or specific configuration of your device model in the Google Nearby Devices console.
-These requirements typically help to improve user experience or security properties for the chosen use case.
-To learn more, see the official `Fast Pair`_ documentation.
+The `Fast Pair Device Feature Requirements`_ category in the official `Fast Pair`_ documentation defines additional requirements for each supported use case.
```
Comment (annwoj) [inline doc/nrf/external_comp/bt_fast_pair.rst:846]:
```suggestion
In the official `Fast Pair`_ documentation, the `Fast Pair Device Feature Requirements`_ category defines additional requirements for each supported use case, and specifies a list of mandatory, optional, and unsupported Fast Pair features.
```

File: doc/nrf/external_comp/bt_fast_pair.rst [hunk 8]
```diff
@@ -843,9 +843,11 @@ Tailoring protocol for specific use case
...
+Each listed use case specifies a list of mandatory, optional and unsupported Fast Pair features.
```
Comment (annwoj) [inline doc/nrf/external_comp/bt_fast_pair.rst:847]:
```suggestion
```

**Review by annwoj — APPROVED** [review]
Just last suggestion to make it flow better - approving

**Pull request merged**